frontend/src/components/InlineMenu/Menu.tsx

```diff
@@ -46,7 +46,8 @@ export const Menu: React.FC<MenuProps> = ({
   const { contentClassName, itemClassName, expandedClassName, groupClassName } = useMemo(() => {
     if (
       menuItems.some(
-        ({ children, thumbnail }) => (children && children.length > 5) || thumbnail !== null,
+        ({ children }) =>
+          children && (children.length > 5 || children.some(({ thumbnail }) => thumbnail !== null)),
```

Comment:
```suggestion
        ({ children = [] }) =>
          (children.length > 5 || children.some(({ thumbnail }) => thumbnail !== null)),
```

Reply:
Good point, I'll take it and delete the extra brackets :+1: 
```suggestion
        ({ children = [] }) =>
          children.length > 5 || children.some(({ thumbnail }) => thumbnail !== null),
```